src/NetworkCanvas.js

```diff
@@ -192,7 +192,7 @@ export default class NetworkCanvas extends EventEmitter {
   }
 
   onLeaveAnnotation = () =>  {
-    this.currentHover.remove();
+    this.currentHover?.remove();
     this.currentHover = null;
   }
```

Comment:
Out of interest: this should only get called if `onLeaveAnnotation` happens without `onEnterAnnotation` first, right? What are the cases where this occurs? Does this have to do with scrolling?

Reply:
I didn't debug it in detail but I observed that sometimes the hover would not be there when scrolling out. I guess it could happen if the screen was re-rendered between the time the mouse entered and left the annotation.